### DISCUSSION TITLE
Dashed line

### DISCUSSION THREAD

**Discussion by Gossenruda** [Live-Charts/LiveCharts2#307] [Q&A] [answered]
I would like to place an alarm limit with a dashed line in a chart. I have not managed to find any property for line appearance, does anyone know if this is possible?

**Comment by jnayee** [comment, marked as answer]
A DashEffect can be applied to the Stroke as shown below:

`    public IEnumerable<ISeries> Series { get; set; }
        = new ISeries[] {
            new LineSeries<int>
            {
                Values = new int[] { 5, 3, 2, 8, 7, 4, 5, 4, 6, 7, 5 },
                Fill = null,
                GeometrySize = 0,
                Stroke = new SolidColorPaint  {
                    Color = SKColors.Red,
                    StrokeThickness = 2,
                    PathEffect = new DashEffect(new float[] { 3, 3 })
                }
            }
        };
`
